### PR TITLE
Fix some simple code coverage problems

### PR DESCRIPTION
**Pull request by yufeih**
- Remove debug only ToString() implementations
- Added some missing tests
- Some simple cleanup

### CODE REVIEW

File: test/docfx.Test/lib/ParallelUtilityTest.cs [hunk 1]
```diff
@@ -20,7 +20,12 @@ public static async Task ThrowsTheSameException(Type exceptionType)
             var exception = await Assert.ThrowsAnyAsync<Exception>(() => ParallelUtility.ForEach(Enumerable.Range(0, 1000), Run));
             Assert.Equal(exceptionType, exception.GetType());
 
+            exception = await Assert.ThrowsAnyAsync<Exception>(() => ParallelUtility.ForEach(Enumerable.Range(0, 1000), RunExpandChild));
+            Assert.Equal(exceptionType, exception.GetType());
+
             Task Run(int n) => n % 500 == 0 ? throw (Exception)Activator.CreateInstance(exceptionType) : Task.CompletedTask;
+
+            Task RunExpandChild(int n, Action<int> queue) => n % 500 == 0 ? throw (Exception)Activator.CreateInstance(exceptionType) : Task.CompletedTask;
```
Comment (live1206) [inline test/docfx.Test/lib/ParallelUtilityTest.cs]: What is the difference between `RunExpandChild` and `Run`? `Action<int> queue` is not in use

File: test/docfx.Test/lib/ParallelUtilityTest.cs [hunk 2]
```diff
@@ -20,7 +20,12 @@ public static async Task ThrowsTheSameException(Type exceptionType)
             var exception = await Assert.ThrowsAnyAsync<Exception>(() => ParallelUtility.ForEach(Enumerable.Range(0, 1000), Run));
             Assert.Equal(exceptionType, exception.GetType());
 
+            exception = await Assert.ThrowsAnyAsync<Exception>(() => ParallelUtility.ForEach(Enumerable.Range(0, 1000), RunExpandChild));
```
Comment (live1206) [inline test/docfx.Test/lib/ParallelUtilityTest.cs]: Shall we put this in a separate test? If the test fails It would be hard to see which case is failing

Comment (yufeih) [reply test/docfx.Test/lib/ParallelUtilityTest.cs]: then all the `[TheoryData]` would be duplicated

Comment (live1206) [reply test/docfx.Test/lib/ParallelUtilityTest.cs]: yes, the duplication of `[TheoryData]` should be fine

**Pull request merged**